guides/security/samesite-protection.md

```diff
@@ -0,0 +1,28 @@
+# SameSite protection
```

Comment:
The guide should be placed inside `guides/plugins/plugins/storefront/*`, because it only relates to plugins and not to apps or themes.

Reply:
@elkmod shouldn't this be placed somewhere more general since it does affect how cookies are handled across the whole system?

Reply:
In that case it is maybe better placed inside the hosting section, but not under guides?